### ISSUE TITLE
Link sitemaps using gatsby-plugin-sitemap

### ISSUE THREAD

**Issue by Kosai106** [gatsbyjs/gatsby#15860]
## Summary

TL;DR - Is it possible to link two sitemaps using `gatsby-plugin-sitemap` as described [here](https://support.google.com/webmasters/answer/75712?hl=en)?

## Relevant information

I have two Gatsby applications - One for my main website, and another for a specific sub-path. Sadly there's no way I can merge these apps, I need them to be separate.

To improve our SEO, we'll need a sitemap for each, which can of course easily be generated in each project, however I can't seem to find a way to have them merged in the main `sitemap.xml`. Have anyone come across a similar situation and are able to help, that would be much appreciated.

## Example

Main sitemap:
```xml
<?xml version="1.0" encoding="UTF-8"?>
  <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:news="http://www.google.com/schemas/sitemap-news/0.9" xmlns:xhtml="http://www.w3.org/1999/xhtml" xmlns:mobile="http://www.google.com/schemas/sitemap-mobile/1.0" xmlns:image="http://www.google.com/schemas/sitemap-image/1.1" xmlns:video="http://www.google.com/schemas/sitemap-video/1.1">
    <url>
      <loc>https://www.example.com/</loc>
      <changefreq>daily</changefreq>
      <priority>0.7</priority>
    </url>
  </urlset>
```

Subpath sitemap:
```xml
<?xml version="1.0" encoding="UTF-8"?>
  <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:news="http://www.google.com/schemas/sitemap-news/0.9" xmlns:xhtml="http://www.w3.org/1999/xhtml" xmlns:mobile="http://www.google.com/schemas/sitemap-mobile/1.0" xmlns:image="http://www.google.com/schemas/sitemap-image/1.1" xmlns:video="http://www.google.com/schemas/sitemap-video/1.1">
    <url>
      <loc>https://www.example.com/products/item1</loc>
      <changefreq>daily</changefreq>
      <priority>0.7</priority>
    </url>
  </urlset>
```

Desired linked sitemap
```xml
<?xml version="1.0" encoding="UTF-8"?>
   <sitemapindex xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
   <sitemap>
      <loc>http://www.example.com/sitemap.xml</loc>
      <lastmod>2019-07-17</lastmod>
   </sitemap>
   <sitemap>
      <loc>http://www.example.com/products/sitemap.xml</loc>
      <lastmod>2019-07-16</lastmod>
   </sitemap>
   </sitemapindex>
```

**Comment by Kosai106** [comment]
So instead, would you recommend me to generate the sitemaps as I normally would, then using `onPostBuild` to write a node function that'll create the linked sitemap for me instead, or do you have any other recommendations?

**Comment by freiksenet** [comment]
@Kosai106 That sounds like it would work as a workaround :+1: You could also make a PR to the plugin to support that kind of sitemap generation or copy the plugin to your local plugins and update it there.

**Comment by Kosai106** [comment]
I think it's fair to close this one.
